components/Dropdown/Dropdown.less

```diff
@@ -17,9 +17,10 @@
     width: 100%;
     position: absolute;
     background-color: @white;
-    display: flex;
+    display: none; // GSAP
+    opacity: 0; // GSAP
```

Comment:
good eye, controlling the default styles is key to getting the desired behavior with GSAP